libs/ui/src/checkbox_button.tsx

```diff
@@ -6,6 +6,7 @@ export interface CheckboxButtonProps {
   isChecked: boolean;
   onChange: (isChecked: boolean) => void;
   disabled?: boolean;
+  style?: React.CSSProperties;
```

Comment:
Consider reverting this if you do the `styled(CheckboxButton)` as I suggested earlier.